spec/defines/rule_spec.rb

```diff
@@ -833,5 +765,89 @@
     it do
       is_expected.to contain_file('/etc/logrotate.d/btmp').without_content(%r{/ifempty/})
     end
+
+    it do
+      is_expected.to contain_file('/etc/logrotate.d/foo').with(
+        'owner' => 'root',
+        'group' => 'root',
+        'ensure' => 'present',
+        'mode' => '0644'
+      )
+    end
+  end
+
+  context 'hourly rules' do
+    let(:title) { 'btmp' }
+    let(:params) do
+      {
+        path: '/var/log/btmp',
+        rotate_every: 'hourly'
+      }
+    end
+    let(:facts) do
+      {
+        os: { family: 'RedHat', release: { major: '7' } },
+        operatingsystemmajrelease: 7
```

Comment:
`operatingsystemmajrelease` is a legacy fact, not needed anymore.

Reply:
removed!